.github/workflows/test.yml

```
@@ -88,14 +116,134 @@
+            echo "RELEASE_TESTS_FAILED=false" >> $GITHUB_ENV
+          fi
 
-  shellcheck:
-    name: Shellcheck
+  ha-core-dev:
     runs-on: ubuntu-latest
+    name: Setup for HA-core (dev)
+    needs:
+      - pre-commit
+    outputs:
+      dev_failed: ${{ steps.ha_core_tests.outputs.failed }}
```

actionlint (1.7.4)

property "ha_core_tests" is not defined in object type {cache-hacore: {conclusion: string; outcome: string; outputs: {string => string}}; determine_mode: {conclusion: string; outcome: string; outputs: {string => string}}; ha-core-tests: {conclusion: string; outcome: string; outputs: {string => string}}; python: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)